### PR TITLE
V8 release 8.6

### PR DESCRIPTION
**Pull request by RReverser**

### CODE REVIEW

File: src/blog/v8-release-86.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: 'V8 release v8.6'
+author: 'Ingvar Stepanyan, a keyboard fuzzer'
```
Comment (mathiasbynens) [inline src/blog/v8-release-86.md:3]: feel free to link to your Twitter here, like in other posts (unless you prefer not to)

File: src/blog/v8-release-86.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+avatars:
+ - 'ingvar-stepanyan'
+date: 2020-09-21
+tags:
+ - release
+description: 'V8 release v8.6 brings respectful code, performance improvements and normative changes.'
```
Comment (mathiasbynens) [inline src/blog/v8-release-86.md:9]: Oxford comma!!1 (We match `Intl.ListFormat` here, hehe)

```suggestion
description: 'V8 release v8.6 brings respectful code, performance improvements, and normative changes.'
```

File: src/blog/v8-release-86.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+Converting a JavaScript number to a string can be a surprisingly complex operation in the general case; we have to take into account floating point precision, scientific notation, NaNs, infinities, rounding, and so on. We don’t even know how big the resulting string will be before calculating it. Because of this, our implementation of `Number.prototype.toString` would bail out to a C++ runtime function.
+
+But, a lot of the time, you just want to print a simple, small integer (a “Smi”). This is a much simpler operation, and the overheads of calling a C++ runtime function are no longer worth it. So we’ve worked with our friends at Microsoft to add a simple fast path for small integers to `Number.prototype.toString`, written in Torque, to reduce these overheads for this common case. This improved number printing microbenchmarks by ~75%.
+
+### `Atomics.wake` removed
+
+`Atomics.wake` was renamed to `Atomics.notify` to match a spec change [in v7.3](https://v8.dev/blog/v8-release-73#atomics.notify). The deprecated `Atomics.wake` alias is now removed.
+
+### Small Normative Changes
```
Comment (mathiasbynens) [inline src/blog/v8-release-86.md:37]:
```suggestion
### Small normative changes
```

File: src/blog/v8-release-86.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Up until now, SIMD was implemented only in TurboFan, V8's top tier compiler. This is necessary to get maximum performance out of the SIMD instructions. Now that SIMD is implemented in Liftoff as well, you'll be able to debug your SIMD code in Liftoff, with much [better performance than the WebAssembly interpreter](https://v8.dev/blog/v8-release-84#better-debugging).
+
+WebAssembly modules that use SIMD instructions will have faster startup too, and often faster runtime performance than their scalar equivalents compiled with TurboFan. For example, given a function that takes an array of floats and clamps its values to zero (written here in JavaScript for clarity):
+
+```js
+function clampZero(f32array) {
+    for (let i = 0; i < f32array.length; ++i) {
```
Comment (mathiasbynens) [inline src/blog/v8-release-86.md:55]: nit: should be two-space indents

File: src/blog/v8-release-86.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+Let's compare two different implementations of this function, using Liftoff and TurboFan:
+
+1. A scalar implementation, with the loop unrolled 4 times.
+2. A SIMD implementation, using the `i32x4.max_s` instruction.
+
+Using the Liftoff scalar implementation as a baseline, we see the following results:
+
+![A graph showing Liftoff SIMD being ~2.8x faster than Liftoff scalar vs. Turbofan SIMD being ~7.5x faster](/_img/v8-release-86/simd.svg)
```
Comment (mathiasbynens) [inline src/blog/v8-release-86.md:70]:
```suggestion
![A graph showing Liftoff SIMD being ~2.8× faster than Liftoff scalar vs. TurboFan SIMD being ~7.5× faster](/_img/v8-release-86/simd.svg)
```

**Review by mathiasbynens — APPROVED** [review]
LGTM % nits

File: src/blog/v8-release-86.md (hunk 1 above)
Comment (RReverser) [reply src/blog/v8-release-86.md:3]: Ah right. Been a while :P

Comment (RReverser) [reply src/blog/v8-release-86.md:3]:
```suggestion
author: '[Ingvar Stepanyan](https://twitter.com/RReverser), a keyboard fuzzer'
```

**Pull request merged**